tools/src/main/java/org/apache/kafka/tools/consumer/group/CsvUtils.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.tools.consumer.group;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+
+public class CsvUtils {
+    private final CsvMapper mapper = new CsvMapper();
+
+    ObjectReader readerFor(Class<? extends CsvRecord> clazz) {
+        return mapper.readerFor(clazz).with(getSchema(clazz));
+    }
+
+    ObjectWriter writerFor(Class<? extends CsvRecord> clazz) {
+        return mapper.writerFor(clazz).with(getSchema(clazz));
+    }
+
+    private CsvSchema getSchema(Class<? extends CsvRecord> clazz) {
+        String[] fields;
+        if (CsvRecordWithGroup.class == clazz)
+            fields = CsvRecordWithGroup.FIELDS;
+        else if (CsvRecordNoGroup.class == clazz)
+            fields = CsvRecordNoGroup.FIELDS;
+        else
+            throw new IllegalStateException("Unhandled class " + clazz);
+
+        return mapper.schemaFor(clazz).sortedBy(fields);
+    }
+
+    public interface CsvRecord {
```

Comment:
Do we need this dumb interface? We use `ObjectWriter` to output strings and it can take `Object` directly.

Reply:
Done